tests/test_base.py

```diff
@@ -15,6 +16,11 @@
 except ImportError:
     import mock
 
+if sys.version_info.major != 3:
```

Comment:
this is very obscure, can you explain why are you adding these lines?
Why are these needed? Is urlparse import failing somewhere?

Reply:
Coveralls is only failing on python2 as it does not reach that exception, therefore decreasing the coverage

Reply:
That's not the way to do it.
Please add `#noqa` if you don't want coveralls to decrease your coverage! :smile: 

Thanks!